app/src/main/java/kr/ac/kaist/team888/core/Stroke.java

```diff
@@ -178,13 +182,13 @@ public StrokeBuilder setEndPoint(Point2D endPoint) {
       return this;
     }
 
-    public StrokeBuilder setControlPoint(Point2D controlPoint) {
-      this.controlPoint = controlPoint;
+    public StrokeBuilder addControlPoint(Point2D controlPoint) {
```

Comment:
Suggest `addControlPoints(ArrayList<Point2D> controlPoints)`.
I think it is more proper for the current data structure.